.github/workflows/python-package.yml

```diff
@@ -23,14 +23,19 @@ jobs:
       run: |
         python -m pip install --upgrade pip
         python -m pip install flake8 pytest
-        pip install -r requirements.txt
+        pip install .
```

Comment:
I would prefer explicit file handling...

Reply:
you mean reverting to the previous command? That way only the requirements are installed but not the package itself, so I wouldn't be able to test scIB import

Reply:
I am sure that I uninstall the scIB and try to reinstall it, but I still cannot import metric. You can also try this on colab by yourself.

Reply:
Ah, sorry... okay. I wasn't thinking clearly ^^.